src/content/1.7/modules/concepts/controllers/admin-controllers.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Admin controllers
+weight: 3
+---
+
+# Create Admin module controllers
+
+> Available since **PrestaShop** {{< minver v="1.7.5" >}}
+
+* Since 1.7.3 you create and override templates and services in your modules.
+* Since 1.7.4, you can create and override forms and console commands.
+* Since 1.7.5, you can create your own "modern" controllers!
+
+## What does it means?
+
+This means you can rely on modern environment to add new entry points to your applications.
+Using modern pages, you will have access to the PrestaShop debug toolbar, the service container, Twig and Doctrine.
+For your views, the PrestaShop UI Kit is available, on top of Bootstrap 4 and ensuring your views are consistent with the PrestaShop Back Office.
+
+## How to declare a new Controller?
+
+Somewhere in your module declare a new class that will act as a Controller:
+
+```php
+// modules/your-module/controller/DemoController.php
+
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+
+class DemoController extends FrameworkBundleAdminController
```

Comment:
Maybe write something about AdminSecurity?

Reply:
it's documented in migration guide